### DISCUSSION TITLE
Security group for the environment

### DISCUSSION THREAD

**Discussion by javiertafurpino** [microsoft/powerplatform-build-tools#607] [Q&A]
Do exist a way to assign a security group to the environment in the moment of the creation using the command pac admin create? Or maybe other way?

**Comment by petrochuk** [comment]
`pac admin create` will have `--security-group-id` in next release which should be out in about two weeks

**Reply by rpothin** [reply]
This is great news @petrochuk! Thank you very much for your work on that 👍🏼

**Reply by javiertafurpino** [reply]
Tears of happiness on my eyes 🥲. Good news . Thanks.
